benchmarks/tpc-ds/sf1.json

```diff
@@ -1,5 +1,5 @@
 {
   "description": "Specifies the catalog and schema for TPC-DS scale factor 1. For this scale factor we use the TPC-DS connector.",
-  "catalog": "tpcds",
-  "schema": "sf1"
+  "catalog": "hive",
+  "schema": "tpcds"
```

Comment:
I don't know if you wanted to check in this change? We wanted sf1 to be from the tpcds connector so people can easily try this out in their local set up.

Reply:
Yep that was my fault let me change it back.